backend/static/util.py

```diff
@@ -0,0 +1,39 @@
+from os import remove
+from os.path import exists
+
+from flask import current_app
+
+from static.exception import FileNotExistError
+
+
+def is_image_with_specific_id_exist(image_id: str) -> bool:
+    image_path: str = _get_file_path_by_image_id(image_id)
+    return exists(image_path)
+
+
+def delete_image(image_id: str) -> None:
+    image_path: str = _get_file_path_by_image_id(image_id)
+    if not is_image_with_specific_id_exist(image_id):
+        raise FileNotExistError(image_path)
+    else:
+        remove(image_path)
+
+
+def put_image(image_data: str, image_id: str):
+    _write_file(image_data, image_id)
```

Comment:
`_write_file` has an identical signature with `put_image`, which seems unnecessary. I suggest you keep the `write_file` function, and we can have the image putting route call it in the future.